scripts/secret_manager.py

```diff
@@ -0,0 +1,76 @@
+import os
+import json
+import argparse
+
+import boto3
+
+# Get Current Directory
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+PARENT_DIR = os.path.dirname(CURRENT_DIR)
+FILE_NAME = ".env"
+
+FILE_PATH = os.path.join(PARENT_DIR, FILE_NAME)
+
+
+def main(arg):
+    client = boto3.client("secretsmanager")
+    secrets_with_values = get_secrets_with_values(client)
+
+    if arg.update or not os.path.exists(FILE_PATH):
+        print(f"Writing secrets to {FILE_PATH}")
+        with open(FILE_PATH, "w") as f:
+            for key, value in secrets_with_values.items():
+                f.write(f"{key}={value}\n")
+    else:
+        print("File already exists. Use --update or make update_secrets to overwrite.")
+
+
+def get_secrets_with_values(client):
+    secrets = {}
+
+    next_token = ""
+    testing_only_secrets = [
+        {
+            "Key": "name",
+            "Values": [
+                "testing/",
+            ],
+        }
+    ]
+
+    while True:
+        if not next_token:
+            response = client.batch_get_secret_value(
+                Filters=testing_only_secrets,
+            )
+        else:
+            response = client.batch_get_secret_value(
+                Filters=testing_only_secrets,
+                NextToken=next_token,
+            )
+
+        for s in response["SecretValues"]:
```

Comment:
Do we need to check the response for success or use `get("SecretValues")` and then check on whether that's not to avoid a key error?